test/memory.test.js

```diff
@@ -548,6 +548,42 @@ describe('Memory connector', function() {
     });
   });
 
+  describe('autoupdate', function() {
+    var ds, model;
+    beforeEach(function() {
+      ds = new DataSource({
+        connector: 'memory'
+      });
+
+      model = ds.createModel('m1', {
+        name: String
+      });
+
+      ds.automigrate();
+
+      ds.createModel('m1', {
+        name: String,
+        address: String
+      });
+    });
+
+    it.skip('autoupdates all models', function(done) {
+      ds.autoupdate(function(err, result){
+        done(err);
+      });
+    });
+
+    it.skip('autoupdates all models - promise variant', function(done) {
+      ds.autoupdate()
+        .then(function(result) {
+            done();
+          })
+        .catch(function(err){
+          done(err);
+        });
+    });
+  });
+
```

Comment:
I was planning to add test cases for `autoupdate`, using -

```
ds = new DataSource({
  connector: 'memory'
});
```
But then realized that memory connector does not have `autoupdate` method. Hence skipped both the test cases above. How should I be verifying `promise` support for `autoupdate`?